pkg/apis/cyclone/v1alpha1/workflow.go

```diff
@@ -28,3 +28,10 @@ type StageItem struct {
 	// Stages that this stage depends on
 	Depends []string `json:"depends"`
 }
+
```

Comment:
:point_up_2: 

```go
// WorkflowSpec defines workflow specification.
type WorkflowSpec struct {
	Stages StageItem `json:"stageItem"`
}
```
should be:

```go
// WorkflowSpec defines workflow specification.
type WorkflowSpec struct {
	Stages []StageItem `json:"stageItem"`
}
```
?

Reply:
Also, can we change json tag `stageItem` to just `stages`?

Reply:
```
type WorkflowSpec struct {
	Stages []StageItem `json:"stages"`
}
```

You are right.